RollingPaper/RollingPaper/ViewControllers/Core/Setting/AppSettingViewController.swift

```diff
@@ -249,86 +286,18 @@ final class AppSettingViewController: UIViewController, UICollectionViewDelegate
-        collectionView.delegate = self
-        
-        userInfoStack.snp.makeConstraints { make in
-            make.leading.equalTo(view.safeAreaLayoutGuide).offset(16)
-            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(16)
-            make.height.equalTo(150)
-            make.top.equalTo(view.safeAreaLayoutGuide)
-        }
-        
-        userPhoto.snp.makeConstraints { make in
-            make.height.equalTo(userInfoStack.snp.height).inset(10)
-            make.width.equalTo(userPhoto.snp.height)
-        }
-        
-        collectionView.snp.makeConstraints { make in
-            make.leading.equalToSuperview()
-            make.trailing.equalToSuperview()
-            make.bottom.equalToSuperview()
-            make.top.equalTo(userInfoStack.snp.bottom)
-        }
-        
-        userNameStack.layoutMargins = UIEdgeInsets(top: 50, left: 0, bottom: 50, right: 0)
-    }
-}
-
-extension AppSettingViewController {
-    private func toggleSwitchConfiguration() -> UICellAccessory.CustomViewConfiguration {
-        lazy var toggleSwitch: UISwitch = {
-            let toggleSwitch = UISwitch(frame: .zero)
-            toggleSwitch.isOn = false
-            toggleSwitch.addTarget(self, action: #selector(toggleSwitch(sender: )), for: .valueChanged)
-            return toggleSwitch
-        }()
-        
-        lazy var toggleAccessory = UICellAccessory.CustomViewConfiguration(
-            customView: toggleSwitch,
-            placement: .trailing(),
-            isHidden: false
-        )
-        
-        return toggleAccessory
-    }
-    
-    private func colorSelectConfiguration() -> UICellAccessory.CustomViewConfiguration {
-        let colorSelectButton: UISegmentedControl = {
-            let colorSelectButton = UISegmentedControl(items: ["Light", "Dark", "Custom"])
-            colorSelectButton.backgroundColor = UIColor.systemGray4
-            colorSelectButton.tintColor = UIColor.black
-            return colorSelectButton
-        }()
-        
-        lazy var colorSelectAccessory = UICellAccessory.CustomViewConfiguration(
-            customView: colorSelectButton,
-            placement: .trailing(),
-            isHidden: false,
-            reservedLayoutWidth: .actual,
-            maintainsFixedSize: true
-        )
-        
-        return colorSelectAccessory
-    }
-}
-
-class SettingContentCell: UICollectionViewListCell {
-    override func updateConfiguration(using state: UICellConfigurationState) {
-        super.updateConfiguration(using: state)
-        
-        guard var backgroundConfig = self.backgroundConfiguration?.updated(for: state) else { return }
-        backgroundConfig.backgroundColorTransformer = UIConfigurationColorTransformer { _ in
-            state.isSelected || state.isHighlighted ? .systemBackground : .systemBackground
+    @objc private func didChangeValue(segment: UISegmentedControl) {
+        if segment.selectedSegmentIndex == 0 {
```

Comment:
```suggestion
        case segment where segment.selectedSegmentIndex == 0 {
```
등으로 나중에 바꾸면 조금 더 보기 편할수도 있을듯요.